.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Please remove .vscode files and add them to the .gitignore file.

Reply:
@StarpTech done